updates/31/asset_collections_mappings_updates.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO multiasset_mappings(collection_id, asset) VALUES (336, "BTC");
```

Comment:
check the others but I believe is two rows per entry

```suggestion
INSERT INTO multiasset_mappings(collection_id, asset) VALUES (336, "BTC");
*
```